<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Color</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="infLabel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>your color:</string>
    </property>
   </widget>
   <widget class="QLabel" name="pallete">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>340</width>
      <height>180</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="chosencolor">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>90</y>
      <width>121</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>chosing color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vopyButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>380</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy HEX</string>
    </property>
   </widget>
   <widget class="QComboBox" name="choseColor1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>181</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>RGB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CMYK</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HLS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HSV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>XYZ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LAB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="choseColor2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>240</y>
      <width>181</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>RGB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CMYK</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HLS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HSV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>XYZ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LAB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="choseColor3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>240</y>
      <width>181</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>RGB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CMYK</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HLS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HSV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>XYZ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LAB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="SelectColorBotton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>380</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Color</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
